### ISSUE TITLE
Scan for New Music Playlist

### ISSUE THREAD

**Issue by mbukosky** [mbukosky/SpotifyUnchained#2]
This way we don't need to have a blog like https://community.spotify.com/t5/Music-Chat/New-Music-Tuesday-Archive/td-p/969479

**Comment by mbukosky** [comment]
I was able get a download of the New Music Playlist data. This playlist is from the main 'spotify' user account. I was able to use the angular-spotify module to get with the api calls.

example

<pre>{
  "collaborative": false,
  "description": "The best newly released songs packed into one playlist. Updated weekly!",
  "external_urls": {
    "spotify": "http://open.spotify.com/user/spotify/playlist/1yHZ5C3penaxRdWR7LRIOb"
  },
  "followers": {
    "href": null,
    "total": 514462
  },
  "href": "https://api.spotify.com/v1/users/spotify/playlists/1yHZ5C3penaxRdWR7LRIOb",
  "id": "1yHZ5C3penaxRdWR7LRIOb",
  "images": [
    {
      "height": null,
  ......
</pre>
